### PR TITLE
Add JOSS Article, Review badge and Citation information

### LINKED ISSUE #95: [PyOpenSci] Submit xclim to the Journal of Open Source Software
https://joss.theoj.org/about

The Journal of Open Source Software is an initiative for Research Software Engineers (RSEs) who primarily develop tools to facilitate research to be able to showcase their accomplishments. It also provides an easy method to inform other researchers about Ouranos and xclim (or possibly other work we may do). 

The approach I'm suggesting is that once we've finalized enough of the library/docs to merit a 1.0 tagged release, we create a one or two page document describing the library and submit this to the journal. They perform a peer-review process and suggest changes that augment the quality of the docs and utility of the library and, just like a typical journal, if accepted we're provided a DOI, a link on their site, and author credit.

Absolutely not a priority for now but I wanted to open a ticket here so that this is on our radars for maybe Q2 2019.

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] This PR addresses an already opened issue (for bug fixes / features)
  - This PR fixes #95  
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGES.rst has been updated

### What kind of change does this PR introduce?

This PR finalizes the submission and publishing steps for the [Journal of Open Source Software](https://joss.readthedocs.io/en/latest/index.html). The paper is expected to have a length of 250 - 1000 words and demands that the software is *feature-complete*. As such, the submission process should not be started until we have at the very least added all necessary indicators from our backlog and/or stabilized our API (i.e.: v1.0-alpha or release-candidate).

Updates (May 2023): 
With xclim v0.40.0, the software was deemed ready for submission. The review process for JOSS was completed via PyOpenSci (https://github.com/pyOpenSci/software-submission/issues/73), and the final JOSS review was performed in https://github.com/openjournals/joss-reviews/issues/5415. The software (v0.43.0) and paper were published on 18 May 2023 (https://doi.org/10.21105/joss.05415).

### Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -32,45 +32,41 @@ authors:
...
 bibliography: paper.bib
 ---
 
 # Summary
 
-`xclim` is a Python library that facilitates the computation of climate indicators over large, heterogeneous data sets. It is based on `xarray` objects and operations, and relies on community conventions for data format and metadata attributes. `xclim` is meant as a tool to facilitate both research and the delivery of operational climate services and products. In addition to climate indicator calculations, `xclim` also includes utilities for bias correction and statistical adjustment, ensemble analytics and model diagnostics.
+`xclim` is a Python library that enables computation of climate indicators over large, heterogeneous data sets. It is built using `xarray` objects and operations, and relies on community conventions for data formatting and metadata attributes. `xclim` is meant as a tool to facilitate both climate science research and the delivery of operational climate services and products. In addition to climate indicator calculations, `xclim` also includes utilities for bias correction and statistical adjustment, ensemble analytics, and model diagnostics.
 
-# Statement of need
+# Statement of Need
```
Comment (huard) [inline paper/paper.md:45]: The example given does not capitalize sub-titles.

**Comment by huard** [comment]
Note that we don't need to merge this branch:

> Your paper (paper.md and BibTeX files, plus any figures) must be hosted in a Git-based repository together with your software (although they may be in a short-lived branch which is never merged with the default).

**Comment by Zeitsperre** [comment]
There's nothing really stopping us from merging it if we want to, right?

**Comment by huard** [comment]
No, but I`m not sure it's relevant to keep it in master. I suggest that we just tag this branch when submitting.

File: paper/paper.bib [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	month = {jan},
+	publisher = {Wiley},
+	url = {https://doi.org/10.1002%2Fessoar.10510291.1},
+	year = {2022}
+}
+
+@misc{icclim,
+    title = {Python library for climate indices calculation},
+    author = {Christian Pagé and Abel Aoun and Natalia Tatarinova},
+    journal = {GitHub repository},
```
Comment (bzah) [inline paper/paper.bib:42]:
```suggestion
    journal = {GitHub repository},
    doi = {10.5281/zenodo.7382653}
    license = {Apache-2.0 license}
```

**Comment by bzah** [comment]
LGTM, thanks for citing icclim!
Pinging @pagecp for information.

File: paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+The `xclim.ensembles` subpackage bundles utilities to facilitate the analysis of results from multiple models. It includes functions to reduce the ensemble size using clustering algorithms, metrics of ensemble robustness, and significance of climate change signals.
+
+# Projects using `xclim`
+
+`xclim` is core component of Finch [@finch], a server hosting climate analytics services behind a Web Processing Services (WPS) interface. `Finch` itself is part of the computational backend of [climatedata.ca](https://climatedata.ca), an online data portal to access, visualize and analyze climate data over Canada. `xclim` is now also a core component of `icclim` from version 5.0, which itself is used in the `climate4impact` project [@Page:2022].
+
+# Acknowledgements
+
+`xclim` was developed thanks to the financial and strategic contributions of the Canadian Center for Climate Services and the Ouranos Consortium. We acknowledge contributions from Marie-Pier Labonté, David Caron, Jwen-Fai Low, Clair Barnes, Sébastien Biner, Philippe Roy, Carsten Ehbrecht, Tom Keel, Ludwig Lierhammer, Jamie Quinn, Dougie Squire, Ag Stephens, Maliko Tanguy, and Chistopher Whelan.
```
Comment (aulemahal) [inline paper/paper.md:73]:
```suggestion
`xclim` was developed thanks to the financial and strategic contributions of the Canadian Center for Climate Services and the Ouranos Consortium. We acknowledge contributions from Marie-Pier Labonté, David Caron, Jwen-Fai Low, Raquel Alegre, Clair Barnes, Sébastien Biner, Philippe Roy, Carsten Ehbrecht, Tom Keel, Ludwig Lierhammer, Jamie Quinn, Dougie Squire, Ag Stephens, Maliko Tanguy, Jeremy Fyke, Yannick Rousseau, and Chistopher Whelan.
```
Three missing names.

File: paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+## Statistical adjustment and bias correction
+
+The `xclim.sdba` subpackage provides different algorithms to adjust the distribution of simulated variables to observed variables. It adopts a train / adjust paradigm, where corrections are first calculated, then applied to the target data or saved for later use. Most methods support additive or multiplicative corrections, different time groupings (seasonal, monthly, or daily with a rolling window). Correction factors can be interpolated between time groupings to avoid discontinuities in the corrected data.
+
+## Ensemble analysis
+
+The `xclim.ensembles` subpackage bundles utilities to facilitate the analysis of results from multiple models. It includes functions to reduce the ensemble size using clustering algorithms, metrics of ensemble robustness, and significance of climate change signals.
+
```
Comment (aulemahal) [inline paper/paper.md:66]: Should we mention:
-  `analogs`?
- `dataflags` ?
- the `cli` ?

Comment (Zeitsperre) [reply paper/paper.md:66]: I think we should. We don't need to say much. Maybe two sentences each, max.

**Review by huard — APPROVED** [review]

**Comment by Zeitsperre** [comment]
ATTN: @jwenfai @qwhelan @sbiner @davidcaron @raquelalegre @Balinus @ludwiglierhammer @dougiesquire @marielabonte @Beauprel @JamieJQuinn @cehbrecht @JeremyFyke @Ouranosinc/xclim-core 

This paper is in the final stages of the review (congrats!) I wanted to confirm with you all that your ORCID numbers and affiliations are correct. If everything is fine, please just give a :+1: to this post, otherwise please leave a comment!

Thanks again!

**Comment by clairbarnes** [comment]
@Zeitsperre My ORCID is correct, but I'm no longer at UCL full-time - my primary affiliation is now the Grantham Institute for Climate Change and the Environment, Imperial College London. I'd like to keep UCL as my secondary affiliation if possible though, since I'm still an honorary researcher there.

**Comment by malngu** [comment]
> ATTN: @jwenfai @qwhelan @sbiner @davidcaron @raquelalegre @Balinus @ludwiglierhammer @dougiesquire @marielabonte @Beauprel @JamieJQuinn @cehbrecht @JeremyFyke @agstephens @malngu @clairbarnes @Ouranosinc/xclim-core
> 
> This paper is in the final stages of the review (congrats!) I wanted to confirm with you all that your ORCID numbers and affiliations are correct. If everything is fine, please just give a 👍 to this post, otherwise please leave a comment!
> 
> Thanks again!

Not sure you need my ORCID and affiliation, but in case you do, here it is: https://orcid.org/0000-0002-1516-6834. Affiliation: UK Centre for Ecology and Hydrology, Wallingford, UK. Thanks!

**Comment by Balinus** [comment]
> ATTN: @jwenfai @qwhelan @sbiner @davidcaron @raquelalegre @Balinus @ludwiglierhammer @dougiesquire @marielabonte @Beauprel @JamieJQuinn @cehbrecht @JeremyFyke @agstephens @malngu @clairbarnes @Ouranosinc/xclim-core
> 
> This paper is in the final stages of the review (congrats!) I wanted to confirm with you all that your ORCID numbers and affiliations are correct. If everything is fine, please just give a 👍 to this post, otherwise please leave a comment!
> 
> Thanks again!

Thanks! :)

Here's my ORCID: 0000-0003-1239-7589

**Comment by malngu** [comment]
> > ATTN: @jwenfai @qwhelan @sbiner @davidcaron @raquelalegre @Balinus @ludwiglierhammer @dougiesquire @marielabonte @Beauprel @JamieJQuinn @cehbrecht @JeremyFyke @agstephens @malngu @clairbarnes @Ouranosinc/xclim-core
> > This paper is in the final stages of the review (congrats!) I wanted to confirm with you all that your ORCID numbers and affiliations are correct. If everything is fine, please just give a 👍 to this post, otherwise please leave a comment!
> > Thanks again!
> 
> Not sure you need my ORCID and affiliation, but in case you do, here it is: https://orcid.org/0000-0002-1516-6834. Affiliation: UK Centre for Ecology and Hydrology, Wallingford, UK. Thanks!

Just to let you know, I think I have spotted a typo in one of the names (an extra 's'): Gabriel Rondeau-Genessse

**Comment by malngu** [comment]
> ATTN: @jwenfai @qwhelan @sbiner @davidcaron @raquelalegre @Balinus @ludwiglierhammer @dougiesquire @marielabonte @Beauprel @JamieJQuinn @cehbrecht @JeremyFyke @agstephens @malngu @clairbarnes @Ouranosinc/xclim-core
> 
> This paper is in the final stages of the review (congrats!) I wanted to confirm with you all that your ORCID numbers and affiliations are correct. If everything is fine, please just give a 👍 to this post, otherwise please leave a comment!
> 
> Thanks again!

@Zeitsperre Minor change if still possible: My correct affiliation is "UK Centre for Ecology & Hydrology (UKCEH)". Sorry, hadn't spotted it earlier. Don't worry if too late to change.

**Comment by Zeitsperre** [comment]
@malngu The edit is being added in now. Just under the wire!

**Review by huard — APPROVED** [review]

**Pull request merged**